### PR TITLE
Move to Librarian

### PR DESCRIPTION
**Pull request by loles**

### CODE REVIEW

File: examples/openstack/openstack.py [hunk 1]
```diff
@@ -25,9 +22,7 @@
     import pstats
 
     pr = cProfile.Profile()
-
-
-GIT_PUPPET_LIBS_URL = 'https://github.com/CGenie/puppet-libs-resource'
+    pr.enable()
```
Comment (pigmej) [inline examples/openstack/openstack.py]: ?

Comment (loles) [inline examples/openstack/openstack.py]: PROFILE = False
if PROFILE:

**Comment by pigmej** [comment]
works but with current image:
```
[2015-10-20 13:24:18,858: WARNING/Worker-3] Fetched 33.4 MB in 36s (913 kB/s)
[2015-10-20 13:24:18,858: WARNING/Worker-3] [vagrant@10.0.0.4:22] out:
[2015-10-20 13:24:18,858: WARNING/Worker-3] E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/s/spice/libspice-server1_0.12.4-0nocelt2ubuntu1.1_amd64.deb  404  Not Found [IP: 91.189.92.201 80]
[2015-10-20 13:24:18,858: WARNING/Worker-3] [vagrant@10.0.0.4:22] out:
[2015-10-20 13:24:18,858: WARNING/Worker-3] [vagrant@10.0.0.4:22] out:
[2015-10-20 13:24:18,858: WARNING/Worker-3] E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
[2015-10-20 13:24:18,858: WARNING/Worker-3] [vagrant@10.0.0.4:22] out:
[2015-10-20 13:24:18,858: WARNING/Worker-3]
```

so we need new image for it.

**Comment by pigmej** [comment]
Tested new image uploaded and it works. With temporary fix for limits.

**Pull request merged**